docs/topics/Terms-of-Use.md

```diff
@@ -0,0 +1,23 @@
+# Terms of Use
+
+By using the o!TR API, you agree to abide by these terms of use. The o!TR API is provided publicly, for free, for all users by the osu! Tournament Rating Team ("o!TR Team", "us", "our").
+
+## Scraping
+
+The API must not be used to scrape information. Scraping data, such as tournaments, matches, etc. is forbidden and will result in a permanent revocation of API access. Please see the [data](Data.md) page for more information on conveniently importing our dataset.
```

Comment:
I would say that this counts as a form of abuse and the two of these categories can probably be merged. I would keep the `## Abuse` heading and list scraping as one of the examples of what we consider abuse